frontend/src/components/layout/header/UserControls.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <VFlex
+        align-center
+        justify-end
+        row
+        class="pr-2"
+    >
+        <VBtn
+            icon
+            class="notifications mr-4"
+        >
+            <svg>
+                <use href="/assets/icons/bell.svg#root" />
+            </svg>
+        </VBtn>
+        <VToolbarTitle class="hidden-sm-and-down username mr-6">
+            Hello, <span>{{ user.name }}</span>
+        </VToolbarTitle>
+        <VAvatar>
+            <img src="/assets/images/lady.png">
+        </VAvatar>
+        <VMenu
+            bottom
+            offset-y
+        >
+            <template v-slot:activator="{ on }">
+                <VIcon
+                    class="drop-down"
+                    v-on="on"
+                >
+                    arrow_drop_down
+                </VIcon>
+            </template>
+            <VList flat>
+                <VListItem 
+                    v-for="link in links"
+                    :key="link.text"
+                >
+                    <RouterLink
+                        :to="{ name: link.route }"
+                    >
+                        {{ link.text }}
+                    </RouterLink>
+                </VListItem>
+                <VListItem @click="logout">
```

Comment:
probably better create method for this event. We need not just logout user, but also navigate him to home screen